pom.xml

```diff
@@ -183,6 +183,10 @@
         <dep.objenesis.version>1.3</dep.objenesis.version>
         <dep.slice.version>0.10</dep.slice.version>
         <dep.jmh.version>1.15</dep.jmh.version>
+        <dep.junit.jupiter.version>5.0.0-RC2</dep.junit.jupiter.version>
+        <dep.junit.jupiter.platform.version>1.0.0-RC2</dep.junit.jupiter.platform.version>
+        <dep.junit.version>4.12</dep.junit.version>
+        <dep.junit.vintage.version>${dep.junit.version}.0-RC2</dep.junit.vintage.version>
```

Comment:
The JUnit vintage stuff lets JUnit5 run tests from prior versions of JUnit. I'm fine with dropping it entirely since it's unlikely that we're ever going to want to introduce JUnit tests for older versions of JUnit into the Presto codebase.

I took the version number stuff from
https://github.com/junit-team/junit5-samples/blob/master/junit5-maven-consumer/pom.xml#L12